snmp/datadog_checks/snmp/data/default_profiles/hpe-nimble.yaml

```diff
@@ -0,0 +1,103 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _hp-base.yaml
+sysobjectid:
```

Comment:
Missing vendor metadata?

Reply:
example: https://github.com/DataDog/integrations-core/blob/53cffe451f3e373be676bd74df4a317f06cd4c26/snmp/datadog_checks/snmp/data/default_profiles/avtech.yaml#L4-L8

Reply:
oh my bad, it seems `_hp-base.yaml` already have it.